net/openvpn/files/openvpn.init

```diff
@@ -144,6 +144,7 @@ openvpn_add_instance() {
 	local security="$4"
 	local up="$5"
 	local down="$6"
+	local client=$(grep -q "mode.*server" "/var/etc/$conf" || echo 1)
```

Comment:
maybe we have to check `server .*` line
```
local client=$(grep -q "mode.*server" "/var/etc/$conf" || grep -q "^server .*" "/var/etc/$conf" || echo 1)
```
on some case, `mode server` could be missing but `server .*` exist in config file.